tests/unit/core/binarycodec/types/test_issue.py

```diff
@@ -32,7 +32,6 @@ def test_from_value_mpt(self):
         # Test Issue creation for an MPT amount.
         # Use a valid 48-character hex string (24 bytes) for mpt_issuance_id.
         test_input = {
```

Comment:
I'm making a note for future tasks: We need to add unit tests for the `MPTAmount` type. This file now exclusively tests `MPTCurrency` type.

Reply:
hmm yeah. It appears the other features are indirectly testing `MPTAmount`.